lib/modelMaterialsCommon.js

```diff
@@ -0,0 +1,754 @@
+"use strict";
+var Cesium = require('cesium');
+var WebGLConstants = Cesium.WebGLConstants;
+var defined = Cesium.defined;
+var defaultValue = Cesium.defaultValue;
+
+module.exports = modelMaterialsCommon;
+
+function webGLConstantToGlslType(webGLValue) {
```

Comment:
Changes to both would be fine for now; longer term, Cesium will depend on some files in gltf-pipeline.

Also, for Cesium, I would not move this to a separate file; just make the trivial style changes.